tracpro/settings/base.py

```diff
@@ -333,7 +333,7 @@ def _org_scheduler_task(task_name):
         'delete',  # can delete an object
         'list',  # can view a list of the objects
     ),
-    'orgs.org': ('create', 'update', 'list', 'edit', 'home'),
+    'orgs.org': ('create', 'update', 'list', 'edit', 'home', 'fetch_runs'),
```

Comment:
Can you look in the database after doing the `migrate` to see if the permission is created and added to the Administrators group?